.babelrc

```diff
@@ -1,7 +1,21 @@
 {
-  "presets": ["es2015", "stage-0"],
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": true
+        }
+      }
+    ]
+  ],
   "plugins": [
     "add-module-exports",
-    "lodash"
+    "lodash",
+    "@babel/plugin-proposal-class-properties",
+    ["@babel/plugin-transform-async-to-generator", {
+      "module": "bluebird",
+      "method": "coroutine"
+    }]
```

Comment:
More specifically about Bluebird, why is that needed? What is happening without that plugin and this configuration?

Reply:
What happens if we use `@babel/plugin-transform-async-to-generator` without this configuration, with just the default?

What happens if we skip `@babel/plugin-transform-async-to-generator`? Isn't async/await supported natively in node since version 7?